src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -131,10 +126,21 @@ uct_cuda_ipc_mem_add_reg(void *addr, uct_cuda_ipc_memh_t *memh,
         return UCS_ERR_NO_MEMORY;
     }
 
-    legacy_handle = (CUipcMemHandle*)&key->ph;
     UCT_CUDADRV_FUNC_LOG_ERR(cuMemGetAddressRange(&key->d_bptr, &key->b_len,
                 (CUdeviceptr)addr));
 
+    status = UCT_CUDADRV_FUNC_LOG_ERR(cuPointerGetAttribute(&key->ph.buffer_id,
+                                      CU_POINTER_ATTRIBUTE_BUFFER_ID,
+                                      (CUdeviceptr)addr));
+    if (status != UCS_OK) {
+        goto err;
+    }
+
+    key->ph.handle_type = UCT_CUDA_IPC_KEY_HANDLE_TYPE_ERROR;
+
+    ucs_trace("exporting handle for %p: base %p b_len %lu buffer_id %llu", addr,
+              (void*)key->d_bptr, key->b_len, key->ph.buffer_id);
```

Comment:
remove this log and print buffer id in 233-234 log message - no need two log msgs